### ISSUE TITLE
Windows carriage return in index.js file

### ISSUE THREAD

**Issue by coreydaley** [kovacsv/run-python3#1]
In the currently published version of the run-python3 npm module (v 0.0.4) the index.js file has (at least one) windows carriage return present which causes the following error on MacOS (and probably linux also)

```
corey@Coreys-Macbook-Pro[~/Projects/github.com/kovacsv/Online3DViewer]
> npm run build_engine_prod                                                                                                                                                                                  master

> online-3d-viewer@0.8.23 build_engine_prod
> npm run update_engine_exports && esbuild source/engine/main.js --bundle --minify --global-name=OV --outfile=build/o3dv.min.js


> online-3d-viewer@0.8.23 update_engine_exports
> run-python3 tools/update_engine_exports.py

env: node\r: No such file or directory
```

Would you mind publishing the current master branch (which seems to have a version of 0.0.5) which does not seem to have the same issue when testing it locally and then I would be happy to update the Online3DViewer package.json to use the latest published package.

Thanks!

**Comment by coreydaley** [comment]
@kovacsv Please take a look when you get a moment

**Comment by kovacsv** [comment]
That's kind of strange. The `index.js` file didn't change between 0.0.4 and 0.0.5. Also, the actions for Online3DViewer are running on Windows, MacOS and Linux machines, too. Isn't it possible that it's an anomaly on your computer? What happens if you delete `node_modules` and run `npm install` again?

**Comment by coreydaley** [comment]
I did as you asked (this time on a Fedora Linux machine) and you can see the results here. Before I run `dos2unix` you see that the index.js file has `CRLF` line terminators in it, and afterwards it does not.
```
$ file index.js 
index.js: a /usr/bin/env node script, ASCII text executable, with CRLF line terminators
$ dos2unix index.js 
dos2unix: converting file index.js to Unix format...
$ file index.js 
index.js: a /usr/bin/env node script, ASCII text executable
```

I also ran the file before and after
```
$ ./node_modules/.bin/run-python3 
/usr/bin/env: ‘node\r’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
$ dos2unix node_modules/run-python3/index.js 
dos2unix: converting file node_modules/run-python3/index.js to Unix format...
$ ./node_modules/.bin/run-python3 
Python 3.11.2 (main, Feb  8 2023, 00:00:00) [GCC 12.2.1 20221121 (Red Hat 12.2.1-4)] on linux

```

**Comment by kovacsv** [comment]
To be honest I have no idea what's going on with line endings. Now I switched the npm publisher machine to a linux one instead of a windows one, and published 0.0.5. Could you please check if it works for you now?

**Comment by coreydaley** [comment]
That one works like a charm, thank you sir!
